Makefile

```diff
@@ -104,21 +100,22 @@ install-torch-test:
 	pip install -r examples/torch/requirements.txt
 
 install-torch-dev: install-torch-test install-pre-commit install-pylint
+	pip install -r examples/post_training_quantization/torch/ssd300_vgg16/requirements.txt
+	pip install -r docs/api/requirements.txt
 
 test-torch:
 	pytest tests/common tests/torch --junitxml ${JUNITXML_PATH} $(DATA_ARG)
 
+TORCH_PYFILES := $(shell git ls-files | grep -P ".*torch.*py$$" | grep -v -P "^tools")
+COMMON_PYFILES := $(shell git ls-files | grep -P ".*py$$" \
+					| grep -v -P ".*(torch|tensorflow|onnx|openvino).*" \
+					| grep -v -P "^tools")
+
 pylint-torch:
-	pylint --rcfile .pylintrc               \
-		nncf/common                         \
-		nncf/config                         \
-		nncf/api                            \
-		nncf/torch                          \
-		nncf/experimental/torch             \
-		tests/common                        \
-		tests/torch                         \
-		examples/torch                      \
-		examples/experimental/torch
+	pylint --rcfile .pylintrc   \
+		$(TORCH_PYFILES)        \
+		$(COMMON_PYFILES)
+
```

Comment:
```
pylint-common: 
	pylint --rcfile .pylintrc $(COMMON_PYFILES)
```

Please add `pylint-common` to manual run pylint for common files without install torch requirements, as asked @alexsu52

Reply:
Done, now we should discuss how this will be reflected on CI

Reply:
Agree, done